### PR TITLE
#41:Fixed responsive's bug for course component

### PR DESCRIPTION
**Pull request by ngtrongnguyen**

### CODE REVIEW

File: src/stylesheets/components/_course.scss [hunk 1]
```diff
@@ -1,54 +1,58 @@
...
-            width: 70%;
+            width: 100%;
           }   
         .card{      
           margin: 10px;
+          .course-layout{
+            text-align: start;
+          }
+          @include mobile {
+              width: 14rem;
```
Comment (Lanng21) [inline src/stylesheets/components/_course.scss:27]: 14rem? too large

File: src/stylesheets/components/_course.scss [hunk 2]
```diff
@@ -1,54 +1,58 @@
...
         border: 0.5px solid #B2B3B3;
         border-radius: 16px;
         display: inline-block;
         margin-top: 10px;
         box-sizing: content-box;
         padding: 20px;
+        max-width: 100%;
         @include mobile {
-            width: 70%;
+            width: 100%;
```
Comment (Lanng21) [inline src/stylesheets/components/_course.scss:19]: I don't think it's necessary

File: src/stylesheets/base/_boilerplate.scss [hunk 3]
```diff
@@ -21,4 +21,21 @@ html {
   figure {
     margin: 10px;
   }
+        .container {
```
Comment (Lanng21) [inline src/stylesheets/base/_boilerplate.scss:24]: i think move it to `common`

File: src/components/list-card.html [hunk 4]
```diff
@@ -1,15 +1,21 @@
...
-  <figure>
-    <img src="{{ img }}" alt="pl" class="course-img"/>
+  <figure class="course-layout">
+    <img src="{{ img }}" alt="course's image" class="course-img"/>
   </figure>
-  <div class="texts">
-    <p>{{ content }}</p>
-    <div class="course-symbol">
+  <div class="texts ">
+    <table class="a">
```
Comment (Lanng21) [inline src/components/list-card.html:6]: check naming class

**Comment by Lanng21** [comment]
pls screenshot or video description .

File: src/components/list-card.html [hunk 5]
```diff
@@ -1,15 +1,21 @@
...
+    <table class="course-tabble">
+      <tr>
+        <th>
+           <p class="course-layout">{{ content }}</p>
+        </th>
+      </tr>
+    </table>
+    <div class="course-symbol ">
       <b>4.0</b>
       <div class="Stars" style="--rating: 4;" aria-label="Rating of this product is 4 out of 5."></div>
```
Comment (Lanng21) [inline src/components/list-card.html:15]: i think u should defind variable for rating 
example: `style="--rating: {{rating}}` and reused it for flexibility, not fixed rating

File: src/stylesheets/components/_course.scss [hunk 6]
```diff
@@ -1,54 +1,58 @@
...
         border-radius: 16px;
         display: inline-block;
         margin-top: 10px;
         box-sizing: content-box;
         padding: 20px;
-        @include mobile {
-            width: 70%;
-          }   
+        max-width: 100%;
+        // @include mobile {
```
Comment (Lanng21) [inline src/stylesheets/components/_course.scss:18]: remove

File: src/stylesheets/components/_course.scss [hunk 7]
```diff
@@ -1,54 +1,58 @@
...
+          }
           .texts{
             min-height: 100%;
-            @include tablet {
-              max-height: min-content;
+            .course-symbol{
+              margin-bottom: 1rem ;
+            }
+            th {
+              height: 5rem;
```
Comment (Lanng21) [inline src/stylesheets/components/_course.scss:40]: why u set height here?

**Comment by Lanng21** [comment]
image
check UI

**Comment by Lanng21** [comment]
image
`gap = 4.5rem` too large

**Comment by Lanng21** [comment]
image
check UI footer

**Comment by Lanng21** [comment]
image
compare with design

File: src/stylesheets/components/_joinus-sec.scss [hunk 8]
```diff
@@ -19,6 +20,10 @@
             width: 100%; 
             margin: 0;
           }
+        @include tablet {
+            width: 100%;
```
[the remark this answers is not in the document]
Comment (ngtrongnguyen) [reply src/stylesheets/components/_joinus-sec.scss:24]: Ảnh chụp Màn hình 2023-06-28 lúc 15 20 47

[the remark this answers is not in the document]
Comment (ngtrongnguyen) [reply src/stylesheets/components/_joinus-sec.scss:24]: if i remove it, this UI will display like this

[the remark this answers is not in the document]
Comment (Lanng21) [reply src/stylesheets/components/_joinus-sec.scss:24]: desktop= 100% 
tablet =100%? then repeat that

[the remark this answers is not in the document]
Comment (Lanng21) [reply src/stylesheets/components/_joinus-sec.scss:24]: i mean remove style on mobile

**Review by haunguyen15-tt — APPROVED** [review]

**Review by haunguyen15-tt — APPROVED** [review]

**Pull request merged**